### PR TITLE
refactor: Change method and class import to module imports

### PR DESCRIPTION
**Pull request by MoritzWeber0**
The Google style guide proposes module imports.
In this PR, some old-style imports are changed to the new style.

### CODE REVIEW

File: backend/capellacollab/projects/toolmodels/modelsources/git/routes.py [hunk 1]
```diff
@@ -11,33 +11,29 @@
...
-from capellacollab.projects.toolmodels.modelsources.git.models import (
-    DatabaseGitModel,
-    GitModel,
-    PatchGitModel,
-    PostGitModel,
+from capellacollab.projects.toolmodels import models as toolmodels_models
+from capellacollab.projects.toolmodels.backups import crud as backups_crud
+from capellacollab.projects.toolmodels.modelsources.git import (
+    models as git_models,
 )
```
Comment (Paula-Kli) [inline backend/capellacollab/projects/toolmodels/modelsources/git/routes.py:21]: why are we not importing this as `from . import models` as it is within the same module?

**Review by Paula-Kli — COMMENTED** [review]
good Idea to make this more consistent 👍

File: backend/capellacollab/projects/toolmodels/modelsources/git/routes.py (hunk 1 above)
Comment (Paula-Kli) [reply backend/capellacollab/projects/toolmodels/modelsources/git/routes.py:21]: ok here it is referenced that no relative imports should be used https://google.github.io/styleguide/pyguide.html#224-decision

Comment (MoritzWeber0) [reply backend/capellacollab/projects/toolmodels/modelsources/git/routes.py]: Good catch, thanks for finding it! Is fixed.

**Pull request merged**